src/elements/frame-router.ts

```diff
@@ -118,8 +118,10 @@ export default class FrameRouterElement extends HTMLElement {
         clientInfo && clientInfo.url
       );
 
-      this._frameManager.setFrameSandbox(clientInfo.sandbox);
-      this._frameManager.setFrameAllow(clientInfo.allow);
+      if (clientInfo) {
+        this._frameManager.setFrameSandbox(clientInfo.sandbox);
+        this._frameManager.setFrameAllow(clientInfo.allow);
+      }
```

Comment:
@djhouseknecht clientInfo moves to possibly-null with these changes. I added a conditional around these but I'm rethinking this and wondering if maybe we need to set `sandbox`/`allow` off in the case we can't find clientInfo. Any thoughts?

Reply:
The more I think of it, the more it makes sense to 1) keep the behavior the same and 2) ensure that `sandbox / allow` changes when the route does. So removed the conditional and just pass undefined if there's no clientInfo

Reply:
Yeah, that is exactly the thinking behind that. I was about to comment and then saw your commit. The new null check looks good 👍